### ISSUE TITLE
Line and column doesn't match credo

### ISSUE THREAD

**Issue by ggpasqualino** [iampeterbanjo/vscode-elixir-linter#1]
The line and column highlighted don't match the line and column reported by credo

![image](https://cloud.githubusercontent.com/assets/821260/24881594/91085526-1e3e-11e7-8951-f0b75d07170b.png)

![image](https://cloud.githubusercontent.com/assets/821260/24881520/53512d8e-1e3e-11e7-99ae-c9d1094e58ff.png)

**Comment by iampeterbanjo** [comment]
Thanks for reporting this @ggpasqualino. Do you have a sample file I can use to reproduce this issue? I'm interested in which lines/columns are being reported by the linter and what they should actually be.

**Comment by ggpasqualino** [comment]
it can be as simple as the following
```elixir
defmodule Example do
  def do_it do
  end
end
```

for this credo will say `[R] → lib/course_planner/example.ex:1:11 Modules should have a @moduledoc tag.` but the extension will add the error on line 2, column 12

**Comment by iampeterbanjo** [comment]
Thanks I'll take a look.

**Comment by iampeterbanjo** [comment]
Thanks @ggpasqualino . I've fixed this bug in version 0.2.0 which I have published.
